hack/externalnode/install-vm.sh

```diff
@@ -141,17 +153,280 @@ EOF
+        dependent_packages="ca-certificates curl gnupg lsb-release"
+        cmd="apt-get install -y ${dependent_packages}"
+        execute_command "$cmd" 1m
+        docker_gpg_key="/usr/share/keyrings/docker-archive-keyring.gpg"
+        rm -rf ${docker_gpg_key}
+        docker_download_url="https://download.docker.com/linux/ubuntu"
+        timeout --preserve-status --foreground 10s curl -fsSL ${docker_download_url}/gpg | sudo gpg --dearmor -o ${docker_gpg_key}
+        # Setup a stable repository
+        echo \
+        "deb [arch=$(dpkg --print-architecture) signed-by=${docker_gpg_key}] ${docker_download_url} \
+        $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+        cmd="apt-get update"
+        execute_command "$cmd" 1m
+        docker_install_packages="docker-ce docker-ce-cli containerd.io"
+        cmd="apt-get install -y ${docker_install_packages}"
+        execute_command "$cmd" 2m
+        ;;
+        rhel)
+        # Uninstall older versions of Docker.
+        docker_uninstall_packages="docker docker-client docker-client-latest docker-common docker-latest docker-latest-logrotate docker-logrotate docker-engine podman runc"
+        sudo yum remove ${docker_unistall_packages}
+        cmd="sudo yum install -y yum-utils"
+        execute_command "$cmd" 1m
+        # We are using CentOs Docker images for Rhel as suggested in official website.
+        docker_download_url="https://download.docker.com/linux/centos/docker-ce.repo"
+        cmd="sudo yum-config-manager --add-repo ${docker_download_url}"
+        execute_command "$cmd" 1m
+        docker_install_packages="docker-ce docker-ce-cli containerd.io docker-compose-plugin"
+        cmd="sudo yum install -y ${docker_install_packages}"
+        execute_command "$cmd" 1m
+        cmd="sudo systemctl start docker"
+        execute_command "$cmd" 1m
+        ;;
+    esac
+}
+
+# Install all pre-requisite packages needed for installation.
+install_prerequisite_packages() {
+    set +eo pipefail
+    # Install Docker, docker-compose only when running services in containers.
+    docker version
+    if [[ "$?" -ne 0 ]]; then
+        install_container_runtime
+    fi
+    # Install docker-compose
+    docker-compose version
+    if [ "$?" -ne 0 ]; then
+        cmd="curl -L "${DOCKER_COMPOSE}-$(uname -s)-$(uname -m)" -o ${DOCKER_COMPOSE_DIR}"
+        execute_command "$cmd" 1m
+        if [ "$?" -eq 0 ]; then
+            echo "docker-compose is downloaded successfully"
+            chmod +x ${DOCKER_COMPOSE_DIR}
+            ln -s ${DOCKER_COMPOSE_DIR} /usr/bin/docker-compose
+        else
+            echoerr "Error docker-compose couldn't be downloaded, exiting"
+            exit 2
+        fi
+    fi
+    set -eo pipefail
+}
+
+# Create docker-compose file with OVS and antrea-agent container specific parameters.
+create_docker_compose_file() {
+    cat << EOF > $AGENT_CONF_PATH/docker-compose.yaml
+version: "3"
+
+services:
+  ovs:
```

Comment:
To be consistent, better to use same name as the antrea-agent pod, which is antrea-ovs.

Reply:
Done.